types/magicmirror-module/index.d.ts

```diff
@@ -0,0 +1,83 @@
+// Type definitions for non-npm package magicmirror-module 2.16
+// Project: https://magicmirror.builders/
+// Definitions by: Jalibu <https://github.com/jalibu>
+//                 MichMich <https://github.com/MichMich>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace Module {
+    /* tslint:disable:no-unnecessary-generics */
+    function register<T>(
+        moduleName: string,
+        moduleProperties: ThisType<NonNullable<ModuleProperties<T>>> &
+            Partial<ModuleProperties<T>>,
+    ): void;
+
+    class ModuleProperties<T> {
```

Comment:
This should be a type or interface. We should avoid creating symbols do not exist in the package (why: we've had a bad ride with webpack as a community with `types/webpack`)